lib/active_merchant/billing/gateways/netbanx.rb

```diff
@@ -241,15 +241,11 @@ def headers
         {
           'Accept'        => 'application/json',
           'Content-type'  => 'application/json',
-          'Authorization' => "Basic #{basic_auth}",
+          'Authorization' => "Basic #{Base64.strict_encode64(@options[:api_key].to_s).strip}",
```

Comment:
We can remove `strip` since base64 will not output spaces.

Reply:
👍